src/DataTool.php

```diff
@@ -181,7 +168,32 @@ public function generateData()
             }
         }
     }
-    
+
+    /**
+     * @param $field string - stores field name from fields
+     * @param $fieldDef array - stores data definition for proper field
+     */
+    public function generateFieldData($field, $fieldDef)
+    {
+        $type = (!empty($fieldDef['dbType'])) ? $fieldDef['dbType'] : $fieldDef['type'];
+        $GLOBALS['fieldData'] = $fieldDef;
+
+        /* There are 3 unique parts to the seed: the Module name,
+         * the count of the record, and the name of the field.
+         * Using these 3 things should keep our seed unique enough.
+         */
+        $seed = $this->generateSeed($this->module, $field, $this->count);
+        $value = $this->getData($field, $type, $fieldDef['type'], $seed);
```

Comment:
Actually we can delete `getData` method and place it's body here. This way we can get rid of this weird check below: `if (!empty($value) || $value == '0') {`. What do you think?